custom_model_runner/requirements.txt

```diff
@@ -4,7 +4,6 @@ datarobot>=2.27.0
 trafaret!=1.1.0,<2.0,>=0.7
 docker>=4.2.2,<5.0.0
 flask
-werkzeug==2.0.3
```

Comment:
I would discourage this pin removal and instead suggest updating the hard pin. Based on the RAPTOR-3909 ticket description, I think the following pin would be better suited:
```
werkzeug>=2.1.1,<2.2.0
```

Reply:
thanks...
I'll do `<2.2.0` just in case there are other issues